packages/range/src/ui-range.component.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <label v-if="label" :for="id" :class="theme.cssClass.label">{{
+    label
+  }}</label>
+  <input
+    type="range"
+    v-model="rangeValue"
+    :class="classes"
```

Comment:
since this is only one class you can change this to :class="theme.cssClass.main"